### ISSUE TITLE
Linker Error

### ISSUE THREAD

**Issue by bleft** [adjust/ios_sdk#74]
Using adjust with swift code results in an error.
```
Undefined symbols for architecture x86_64:
  "_ADJEnvironmentSandbox", referenced from: ... 
```
Removing the static keyword from the constants in Adjust.h fixes this issue but results in duplicate symbol errors.

**Comment by nonelse** [comment]
We'll test it and fix it on the next release.
Is any thing else besides swift code that we should know of to replicate it?

**Comment by bleft** [comment]
after using a string instead of the constant everything seems to be working fine.

**Comment by appsunited** [comment]
Is this supposed to be fixed? The issue is still present in the current code base.

**Comment by wellle** [comment]
Thanks for letting us know, we will check that.

**Comment by uerceg** [comment]
Closed with PR #98
